mlflow/tracking/fluent.py

```diff
@@ -1341,7 +1341,8 @@ def setup_autologging(module):
             autolog_fn = LIBRARY_TO_AUTOLOG_FN[module.__name__]
             autologging_params = get_autologging_params(autolog_fn)
             autolog_fn(**autologging_params)
-            _logger.info("Autologging successfully enabled for %s.", module.__name__)
+            if "disable" in autologging_params and not autologging_params["disable"]:
```

Comment:
Nit (also covers the case where `disable` is not present in `autologging_params`):

```suggestion
            if not autologging_params.get("disable", False):
```

Reply:
Oh that's better, thanks